Formula/m/morpheus.rb

```diff
@@ -0,0 +1,98 @@
+class Morpheus < Formula
+  desc "Modelling environment for multi-cellular systems biology"
+  homepage "https://morpheus.gitlab.io/"
+  url "https://gitlab.com/morpheus.lab/morpheus/-/archive/v2.3.7/morpheus-v2.3.7.tar.gz"
+  sha256 "ad5694a098e4752db53659ee983c3ae417a43747320e73c3005f6cf88b52d55c"
+  license "BSD-3-Clause"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+(?:_?\d+)?)$/i)
+  end
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "ninja" => :build
+  depends_on "ffmpeg"
+  depends_on "graphviz"
+  depends_on "libomp"
+  depends_on "libtiff"
+  depends_on "qt@5"
+
+  uses_from_macos "bzip2"
+  uses_from_macos "libxml2"
+  uses_from_macos "zlib"
+
+  def install
+    # has to build with Ninja until: https://gitlab.kitware.com/cmake/cmake/-/issues/25142
+    args = []
+    args << "-G Ninja"
+
+    if OS.mac?
+      args << "-DMORPHEUS_RELEASE_BUNDLE=ON"
+      args << "-DMORPHEUS_SBML=OFF" # SBML import currently disabled due to libSBML build errors with some macOS SDKs
+    end
+
+    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+
+    if OS.mac?
+      bin.write_exec_script "#{prefix}/Morpheus.app/Contents/MacOS/morpheus"
+      bin.write_exec_script "#{prefix}/Morpheus.app/Contents/MacOS/morpheus-gui"
+    end
```

Comment:
```suggestion
    return unless OS.mac?
    
    bin.write_exec_script "#{prefix}/Morpheus.app/Contents/MacOS/morpheus"
    bin.write_exec_script "#{prefix}/Morpheus.app/Contents/MacOS/morpheus-gui"
```

Reply:
Thanks for the suggestion, but we already passed all checks (including style) and according to @chenrui333 we should be ready for merge. I kindly ask that style related matters be modified in the next commit/version bump.

Reply:
Experience teaches me that that postpones it indefinitely. Since this is a suggestion, it should be very easy to incorporate.

Reply:
You need to apply https://github.com/Homebrew/homebrew-core/pull/162155#discussion_r1562266473 anyway, so may as well do this now